File: Stack/stack.py
class Stack:

    def __init__(self, stack_ = []):
        self.stack_ = stack_
        self.top = len(stack_)-1

    def push(self, val):
        self.top += 1
        self.stack_.append(val)

    def display(self):

        for x in range(0 ,self.top):
            print(self.stack_[x - 0])

    def peek(self):

        return self.top


stack = Stack([2,4,5])
stack.push(2)
stack.display()

File: Stack/stack.cpp
#include <iostream>
#include <vector>
using namespace std;
#define size 10

class stack{

    private:
        int stack_[size];
        int top;

    public:

        stack(){
            top = -1;
        }

        void push(int val){
            if( top == size-1 ) cout<<"Stack is full";
            else{
                top++;
                stack_[top] = val;
            }

        }

        void pop(){
            if(top == -1) cout<< "Stack is empty";
            else{
                top--;
            }
        }

        void display(){
            int tmp = top;
            while ( tmp >= 0 ){
                cout<<stack_[tmp];
                tmp--;
            }
        }

        int peek(){
            return stack_[top];
        }


};


int main(){

    stack stk1;
    stk1.push(1);
    stk1.push(2);
    stk1.push(3);
    stk1.push(4);
    stk1.display();

    return 0;
}